### PR TITLE
NO-JIRA minor changes on test

### PR DESCRIPTION
**Pull request by wy96f**

### CODE REVIEW

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java [hunk 1]
```diff
@@ -801,7 +801,6 @@ private void createNewRecord(final long eventUID,
       targetLocator.setInitialConnectAttempts(0);
       targetLocator.setClientFailureCheckPeriod(clientFailureCheckPeriod);
       targetLocator.setConnectionTTL(connectionTTL);
-      targetLocator.setInitialConnectAttempts(0);
```
Comment (clebertsuconic) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java]: what is the outcome of this? faster ? failure? or just to be correct?

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/AddressControlTest.java [hunk 2]
```diff
@@ -282,7 +281,7 @@ public void testGetNumberOfBytesPerPage() throws Exception {
       session.createQueue(address, address, true);
 
       AddressControl addressControl = createManagementControl(address);
-      Assert.assertEquals(ActiveMQDefaultConfiguration.getDefaultJournalFileSize(), addressControl.getNumberOfBytesPerPage());
```
Comment (clebertsuconic) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/AddressControlTest.java]: my previous question also applies here.

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java (hunk 1 above)
Comment (wy96f) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java]: `setInitialConnectAttempts` has been set to 0 two lines before, no need to set it again.

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/AddressControlTest.java (hunk 2 above)
Comment (wy96f) [reply tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/AddressControlTest.java]: It's not semantically right to compare `pageSizeBytes` with default journal file size. The default page size should be AddressSettings.DEFAULT_PAGE_SIZE, see https://github.com/apache/activemq-artemis/blob/eb3c4931c2871a58fea9208eebaf90bc8ffbe462/artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java#L514

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java (hunk 1 above)
Comment (clebertsuconic) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/ClusterConnectionImpl.java]: OIC

**Pull request closed without merging**